[expo-media-library][ios] Add `presentPermissionsPickerAsync` (#10662)
expo/expo: packages/expo-media-library/CHANGELOG.md

### 🛠 Breaking changes

> - Renamed `MediaLibrary.MediaLibraryAssetChangeEvent` type to `MediaLibrary.MediaLibraryAssetChangeEvent`.

Reviewer: ```suggestion
- Renamed `MediaLibrary.MediaLibraryAssetChangeEvent` type to `MediaLibrary.MediaLibraryAssetsChangeEvent`.
```

### 🎉 New features

> - Added the `MediaLibrary.presentLimitedLibraryPickerAsync` function, which will display the system prompt allowing the users to change the selected permitted assets` on iOS.

Reviewer: ```suggestion
- Added the `MediaLibrary.presentPermissionsPickerAsync` method that displays the system prompt allowing the user to change the selected permitted assets` on iOS.
```
